container_pipeline/workers/dockerfile_lint.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+import json
+import logging
+import os
+import subprocess
```

Comment:
Use ``container_pipeline.lib.cmd.run_cmd`` to replace direct interactions with subprocess.